packages/slice-machine/start/index.js

```diff
@@ -160,24 +146,14 @@ async function run() {
   }
 
   const userConfig = handleManifest(cwd);
-  // if (userConfig.state === ManifestStates.NotFound) {
-  //   console.log('Welcome to Slicemachine! 🍕')
-  //   const yes = await shouldOnboard()
-  //   if (!yes) {
-  //     console.log('Please create a config file manually!')
-  //     process.exit(-1)
-  //   } else {
-  //     return start({ cwd, port }, (url) => {
-  //       open(`${url}/onboarding`)
-  //     })
-  //   }
-  // }
-  const { exit, data } = await handleManifestState(userConfig, cwd);
+  const { exit } = await handleManifestState(userConfig, cwd);
   if (exit) {
+    console.log("");
```

Comment:
It for adding a new line  ?

Reply:
yes :) because the exit(-1) writes an error line.
I added for people to not miss our last information like "please create your sm.json file manually"

Reply:
Ohhh ok